ng2-components/ng2-activiti-tasklist/src/components/task-header.component.ts

```diff
@@ -56,24 +57,48 @@ export class TaskHeaderComponent implements OnChanges {
     refreshData() {
         if (this.taskDetails) {
             const parentInfoMap = this.getParentInfo();
+            let assigneeLabel: any;
```

Comment:
Can you please translate the labels using the translate pipe in the html instead of here?
Something like this:
{{ property.label | translate }}

This way we can avoid a lot of unnecessary imperative code.